app/assets/stylesheets/application.css.scss

```diff
@@ -110,6 +110,41 @@ body {
   background-position: center center;
 }
 
+.application-body-map_bree{
```

Comment:
Selector `application-body-map_bree` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space